src/nfs/nfs_server_impl.cpp

```diff
@@ -261,26 +259,11 @@ void nfs_service_impl::register_cli_commands()
 {
     static std::once_flag flag;
     std::call_once(flag, [&]() {
-        _nfs_max_send_rate_megabytes_cmd = dsn::command_manager::instance().register_command(
-            {"nfs.max_send_rate_megabytes_per_disk"},
-            "nfs.max_send_rate_megabytes_per_disk [num]",
-            "control the max rate(MB/s) for one disk to send file to remote node",
-            [](const std::vector<std::string> &args) {
-                std::string result("OK");
-
-                if (args.empty()) {
-                    return std::to_string(FLAGS_max_send_rate_megabytes_per_disk);
-                }
-
-                int32_t max_send_rate_megabytes = 0;
-                if (!dsn::buf2int32(args[0], max_send_rate_megabytes) ||
-                    max_send_rate_megabytes <= 0) {
-                    return std::string("ERR: invalid arguments");
-                }
-
-                FLAGS_max_send_rate_megabytes_per_disk = max_send_rate_megabytes;
-                return result;
-            });
+        _nfs_max_send_rate_megabytes_cmd = dsn::command_manager::instance().register_int_command(
```

Comment:
Should we add a validator for both flag and command to ensure that `max_send_rate_megabytes_per_disk_desc` > 0, since I saw assertions in `src/utils/TokenBucket.h`:
```c++
    boost::optional<double> consumeWithBorrowNonBlocking(double toConsume,
                                                         double rate,
                                                         double burstSize,
                                                         double nowInSeconds = defaultClockNow())
    {
        assert(rate > 0);
        assert(burstSize > 0);
        ......
     }
```

Reply:
It's not needed. 0 (or less than 0) means disable flow control, and 0 is the default value.